### PR TITLE
Expose strict properties

### PR DESCRIPTION
**Pull request by wibisono**
Related to #162 #158

### CODE REVIEW

File: rpki-validator/src/main/resources/application.properties [hunk 1]
```diff
@@ -92,3 +92,8 @@ management.endpoints.enabled-by-default=false
 management.endpoint.prometheus.enabled=true
 management.endpoints.web.exposure.include=prometheus
 
+# Enabling strict validation will not allow stale manifest, stale crl, and reject manifest with non existing entry.
```
Comment (lolepezy) [inline rpki-validator/src/main/resources/application.properties:95]: "with entries referring to non-existent objects"?

File: rpki-validator/src/main/resources/application.properties [hunk 2]
```diff
@@ -92,3 +92,8 @@ management.endpoints.enabled-by-default=false
...
+rpki.validator.strict-validation=false
+
+# Enabling rsync only will ignore RRDP repositories and only use rsync
```
Comment (lolepezy) [inline rpki-validator/src/main/resources/application.properties:98]: Slightly too wordy. 
"This disables RRDP, so the validator only uses rsync to download repositories."?

File: rpki-validator/src/main/resources/application.properties [hunk 3]
```diff
@@ -92,3 +92,8 @@ management.endpoints.enabled-by-default=false
 management.endpoint.prometheus.enabled=true
 management.endpoints.web.exposure.include=prometheus
 
+# Enabling strict validation will not allow stale manifest, stale crl, and reject manifest with entries referring to non-existent objects.
+rpki.validator.strict-validation=false
+
+# This disables RRDP, so the validator only uses rsync to download repositories.
+rpki.validator.rsync-only=false
```
Comment (ties) [inline rpki-validator/src/main/resources/application.properties:99]: Just checking: there is no corresponding rrdp only?

File: rpki-validator/src/main/resources/application.properties [hunk 4]
```diff
@@ -92,3 +92,8 @@ management.endpoints.enabled-by-default=false
 management.endpoint.prometheus.enabled=true
 management.endpoints.web.exposure.include=prometheus
 
+# Enabling strict validation will not allow stale manifest, stale crl, and reject manifest with entries referring to non-existent objects.
```
Comment (ties) [inline rpki-validator/src/main/resources/application.properties:95]: Do stale manifests and crls also get rejected?

Comment (wibisono) [reply rpki-validator/src/main/resources/application.properties:95]: Yes

File: rpki-validator/src/main/resources/application.properties (hunk 3 above)
Comment (wibisono) [reply rpki-validator/src/main/resources/application.properties:99]: Nope.

Comment (lolepezy) [reply rpki-validator/src/main/resources/application.properties:99]: > Just checking: there is no corresponding rrdp only?

We probably don't need "rrdp only" because choosing rrdp is the default behaviour.
And we do it for the testbed, so it should be enough to have a "default" and "rsync only" versions.

Comment (ties) [reply rpki-validator/src/main/resources/application.properties:99]: It does not feel super useful, but it would make it possible to check what goes missing without fallback.

Routinator covers this way of checking.

File: rpki-validator/src/main/resources/application.properties (hunk 4 above)
Comment (ties) [inline rpki-validator/src/main/resources/application.properties:95]:
```suggestion
# Enabling strict validation will reject stale manifest, stale crls, and manifest with entries referring to non-existent objects.
```
